source/_footer.html.erb

```diff
@@ -0,0 +1,7 @@
+<footer>
+    <ul class="footer">
+        <li class="footer__link"><%= link_to '860-406-1021', 'tel:#{8604061021}' %></li>
+        <li class="footer__link"><%= mail_to 'jstolz123@gmail.com', 'jstolz123@gmail.com' %></li>
+        <li class="footer__link"><%= link_to 'Linkedin', 'https://www.linkedin.com/in/jeff-stolz-17666725/', target: "_blank" %></li>
```

Comment:
Both single and double quotes are being used here. We like to stick with double quotes (https://github.com/thoughtbot/guides/tree/master/style/sass). We reserve single quotes for other things like concatenating strings.